guides/sanity-reusable-fields-via-objects.md

```
@@ -0,0 +1,96 @@
+<img src="https://r2cdn.perplexity.ai/pplx-full-logo-primary-dark%402x.png" style="height:64px;margin-right:32px"/>
+
+# in sanity, How can I leverage object definitions to create groups of reusable fields which I can then apply in the schema of multiple document types?
+
+In Sanity, you can leverage **object definitions** to build **reusable groups of fields** that can be shared across multiple document types. This lets you keep your schema DRY (Don’t Repeat Yourself), consistent, and easier to maintain.
+
+### Creating reusable field groups with object definitions
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)